qr-code/node/README.md

```diff
@@ -182,4 +182,7 @@ SHOPIFY_VITE_HMR_USE_POLLING=1 pnpm dev
   - [PHP backend](https://github.com/Shopify/shopify-app-template-php)
   - [Ruby backend](https://github.com/Shopify/shopify-app-template-ruby)
   - [React frontend](https://github.com/Shopify/shopify-frontend-template-react)
+
+```
+
 ```
```

Comment:
What are these?  Artefact of something that has since been deleted?

Reply:
Yup, removing it!